apps/condo/domains/organization/utils/tin.utils.js

```diff
@@ -33,7 +33,7 @@ const isValidTin = (tinValue = null, country = RUSSIA_COUNTRY) => {
     if (country === RUSSIA_COUNTRY) return validateTinRU(tinValue)
 
     // TODO: DOMA-663 add tin validations for countries other than Russian Federation
-    return false
+    return true
```

Comment:
Why?

Reply:
We don’t check non ru inn

Reply:
That's why they're not valid by default :)